python/taichi/lang/matrix.py

```diff
@@ -198,7 +198,7 @@ def _get_slice(self, a, b):
             b = [b]
         else:
             b = range(b.start or 0, b.stop or self.m, b.step or 1)
-        return Matrix([[self(i, j) for j in b] for i in a])
+        return Matrix([[self._get_entry(i, j) for j in b] for i in a])
```

Comment:
Not sure why we need this line of change? Is it trying to support slicing _PyScopeMatrixImpl at ti_scope?

Reply:
No, it's just a bug fix to this function (it complained about `_PyScopeMatrixImpl` is not callable)